scripts/exclude-links.txt

```diff
@@ -0,0 +1,20 @@
+http://127.0.0.1:10000/devstoreaccount1;
+http://localhost:3000/
+https://$
```

Comment:
Hi. Trying to understand how this file is used exactly. Where in the docs is there a lonely `https://` link for example?

Reply:
Or is `$` not the end of a regex match?

Reply:
I guess not because the following expressions `https://api.github.com/repos/$` and `https://blog.$` also wouldn't make sense?

Reply:
they are not, these are exact matches

Reply:
probably answered in https://github.com/iterative/dvc.org/pull/958/files#r374963072

Reply:
> these are exact matches

ah ok thanks Ivan

Reply:
Yes `js` code matches; didn't want to spend too long coming up with a more elegant-yet-problematic solution